### PR TITLE
Cmullenx/angular

### PR DESCRIPTION
**Pull request by cmullenx**
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Please check if the PR fulfills these requirements
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

#### (Bug fix, feature, docs update, ...)

Fixes issues with angular repo not compiling and not being exported into a singular js file for use in desktop

#### (You can link to an open issue)

Angular repo doesnt compile and is not being exported into singular js file for use in desktop

#### (if this is a feature change)?

Fixes issues with angular repo not compiling and not being exported into a singular js file for use in desktop

#### (What changes might users need to make in their application due to this PR?)

`[Type here]`

## Other information:

`[Type here]`

### CODE REVIEW

File: angular/src/app/app.module.ts [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
+//     AppComponent
+//   ],
+//   imports: [
+//     BrowserModule,
+//     AppRoutingModule
+//   ],
+//   providers: [],
+//   bootstrap: [AppComponent]
+// })
+// export class AppModule { }
```
Comment (molker) [inline angular/src/app/app.module.ts:18]: Can take out these comments

**Review by molker — APPROVED** [review]
LGTM
nice job on the fix and angular updates

**Comment by zachraymer** [comment]
Nice work

**Review by zachraymer — APPROVED** [review]

File: angular/README.md [hunk 2]
```diff
@@ -1,17 +1,83 @@
...
+# WxCC Widget Starter - Angular
 
-1. Clone this repo
-2. Run `yarn` from the root of the repo.
-3. Run `yarn start` to start the playground app.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.2.11.
 
-## Disclaimer
+This widget starter can be run on `localhost` to be reviewed. In case you (the developer) are using this widget starter as a foundation of your custom widget, running the sandbox environment on `localhost` would be a perfect way to build your widget's logic and interface.
+
```
Comment (momeraj) [inline angular/README.md:14]: How did we test this widget?
Did we test following scenario?
1. place this widget in the custom page of the desktop
2. place this widget in the aux-panel of the desktop
3. place this widget as header of the desktop
verify if widget getting updated data fromSTORE

**Review by momeraj — CHANGES_REQUESTED** [review]

File: angular/README.md (hunk 2 above)
Comment (molker) [reply angular/README.md:14]: This widget was tested by packaging it and placing it into a custom page in desktop, where it is getting updated data from STORE. 
The other 2 scenarios can be verified

**Pull request merged**